File: eTuristickaAgencija.Models/Termin.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Termin
    {
        public int Id { get; set; }
        public int? DestinacijaId { get; set; }
        public decimal Cijena { get; set; }
        public float? Popust { get; set; }
        public int? HotelId { get; set; }
        public decimal CijenaPopust { get; set; }
        public bool AktivanTermin { get; set; }
        public DateTime DatumPolaska { get; set; }
        public DateTime DatumDolaska { get; set; }
        public int? GradId { get; set; }
    }
}

File: UI/eturistickaagencija_admin/lib/models/termin.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'termin.g.dart';

@JsonSerializable()
class Termin{
int? id;
int? destinacijaId;
double? cijena;
double? popust;
int? hotelId;
double? cijenaPopust;
bool? aktivniTermin;
DateTime? datumPolaska;
DateTime? datumDolaska;
int? gradId;


Termin(this.id,this.destinacijaId,this.cijena,this.popust,this.hotelId,this.cijenaPopust,this.aktivniTermin,this.datumPolaska,this.datumDolaska,this.gradId);

factory Termin.fromJson(Map<String,dynamic>json)=>_$TerminFromJson(json);
Map<String,dynamic>toJson()=>_$TerminToJson(this);
}
